cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -2069,7 +2069,7 @@ macro(build_utf8proc)
       "-DCMAKE_INSTALL_PREFIX=${UTF8PROC_PREFIX}"
       -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
       -DCMAKE_INSTALL_LIBDIR=lib
-      -DDBUILD_SHARED_LIBS=OFF)
+      -DBUILD_SHARED_LIBS=OFF)
```

Comment:
I think this was the original issue, because utf8proc wasn't available from conda, so it was bundled.

Need to run other packaging builds as well if it works.